src/v/cloud_topics/level_zero/gc/level_zero_gc.cc

```diff
@@ -77,6 +82,10 @@ class epoch_source_cluster_impl : public level_zero_gc::epoch_source {
          */
         co_return std::nullopt;
     }
+
+private:
+    [[maybe_unused]] seastar::sharded<cluster::health_monitor_frontend>*
```

Comment:
it is unused because we still must plumb in additional epoch information from the controller before we can take action on GC by computing the final safe to reclaim epoch value.